core-libs/setup/ssr/optimized-engine/optimized-ssr-engine.ts

```diff
@@ -86,30 +87,20 @@ export class OptimizedSsrEngine {
       return;
     }
 
-    const replacer = (_key: string, value: unknown): unknown => {
-      if (typeof value === 'function') {
-        return value.toString();
-      }
-      if (this.isServerLogger(value)) {
-        return value.constructor.name;
-      }
-      return value;
-    };
+    const options = ssrOptimizationOptionsResolver(this.ssrOptions);
```

Comment:
[MINOR]

The name `...Resolver` suggests something more significant than just making options more logger friendly. At the first sight I thought the `...Resolver` will transform the "insufficient" `ssrOptions` and return proper "resolved" options. But we actually need it only for the purpose of logging. What about renaming the function:
ssrOptimizationOptionsResolver -> getLoggableSsrOptimizationOptions
?

e.g.

```suggestion
    const loggableSsrOptions = getLoggableSsrOptimizationOptions(this.ssrOptions);
```